.gitignore

```diff
@@ -197,4 +197,10 @@ src/go/plugin/go.d/bin/
 src/go/plugin/go.d/vendor
 
 # ignore nsis installer
-packaging/utils/netdata-installer.exe
+packaging/utils/netdata-installer-x64.exe
+
+# ignore files used with msi installer
+packaging/windows/*.txt
+packaging/windows/*.rtf
+packaging/utils/netdata.msi
```

Comment:
This should just cover _any_ MSI file, we’re never going to be in a situation where we’re committing any of them, and the name as produced by CI will be including the Windows CPU architecture.